SharpMUSH.Client/wwwroot/css/custom.css

```
@@ -1311,3 +1333,179 @@ body {
+   MOBILE TOOLKIT + RESPONSIVE SHELL
+   Sub-project 0 of the mobile campaign. Everything below the 760px breakpoint.
+
+   Conventions for page authors (Batches 1–4):
+     • Breakpoint is ALWAYS 760px. In .css files: `@media (max-width: 760px)`.
+       In Razor <style> blocks: `@@media (max-width: 760px)` (Razor escapes `@`;
+       a bare `@media` is parsed as a code transition and fails to compile).
+     • Full-height regions: size with `svh` and a `vh` fallback —
+         height: 100vh;   /* fallback for browsers without svh */
+         height: 100svh;  /* fits the SMALL viewport, so it never crops under the
+                             mobile address bar and never resizes on scroll */
+       Default to `svh`, NOT `dvh`. `dvh` tracks the live viewport and recalculates
```

Stylelint (17.13.0)

[error] Unknown word Default (CssSyntaxError)

(CssSyntaxError)